pkg/cmd/roachtest/tests/cdc.go

```diff
@@ -841,11 +906,12 @@ func registerCDC(r registry.Registry) {
 		Cluster:         r.MakeClusterSpec(4, spec.CPU(16)),
 		RequiresLicense: true,
 		Run: func(ctx context.Context, t test.Test, c cluster.Cluster) {
-			ct := newCDCTester(ctx, t, c)
+			ct := newCDCTester(ctx, t, c, true)
 			defer ct.Close()
 
 			ct.runTPCCWorkload(tpccArgs{warehouses: 100})
 
+			ct.setStatsCollectionStartTime()
```

Comment:
Since you have to set a window for stats collection, I decided to create a `start` and `stop` API. The idea is that you call start before starting changefeeds and call export once they finish.